Code/Tools/SceneAPI/SceneUI/CommonWidgets/SceneSettingsCard.cpp

```diff
@@ -280,32 +282,49 @@ void SceneSettingsCard::SetState(State newState)
     switch (newState)
     {
     case State::Loading:
-        setTitle(tr("Loading scene settings"));
-        m_settingsHeader->SetCanClose(false);
+        {
+            QString toolTip(tr("The scene settings for this file are being loaded. The window will update when the event completes."));
+            setTitle(tr("Loading scene settings"));
+            setTitleToolTip(toolTip);
+            m_settingsHeader->m_busySpinner->setToolTip(toolTip);
+            m_settingsHeader->SetCanClose(false);
+        }
         break;
     case State::Processing:
-        setTitle(tr("Saving scene settings, and reprocessing scene file"));
-        m_settingsHeader->SetCanClose(false);
+        {
+            QString toolTip(tr("The scene file is being processed. The window will update when the event completes."));
+            setTitle(tr("Saving scene settings, and reprocessing scene file"));
+            setTitleToolTip(toolTip);
+            m_settingsHeader->m_busySpinner->setToolTip(toolTip);
+            m_settingsHeader->SetCanClose(false);
+        }
         break;
     case State::Done:
         {
+            QString toolTip(tr("No errors or warnings were encountered %1 the scene file.%2"));
             QString errorsAndWarningsString;
             if (m_warningCount > 0 || m_errorCount > 0)
             {
                 errorsAndWarningsString = tr(" with %1 warning(s), %2 error(s)").arg(m_warningCount).arg(m_errorCount);
+                toolTip = tr("Warnings and/or errors were encountered %1 the scene file. You can view the details by expanding this card "
+                             "and reading the log message.%2");
             }
 
             QString previousStateString;
             switch (m_sceneCardState)
             {
             case State::Loading:
                 previousStateString = tr("Loading ");
+                // The second arg is empty because there isn't an easy way to find the loading log messages later.
+                toolTip = toolTip.arg(tr("loading")).arg("");
```

Comment:
from how translations generally go the verb might be contextual depending on the language. not sure if you want to try combines this with the tooltip then trying to split it into arguments like this.